lib/yt/collections/videos.rb

```diff
@@ -93,42 +88,6 @@ def list_params
         end
       end
 
-      def next_page
-        super.tap do |items|
-          halt_list if use_list_endpoint? && items.empty? && @page_token.nil?
-          add_offset_to(items) if !use_list_endpoint? && videos_params[:order] == 'date' && !(videos_params[:for_mine] || videos_params[:for_content_owner])
-        end
-      end
```

Comment:
after removing this we're going to fall back to the definition here: https://github.com/Fullscreen/yt/blob/6e0c51d0f70c03db2f50a6835a1624a30cc473bc/lib/yt/actions/list.rb#L47